servicecontrol/hosting.md

```diff
@@ -0,0 +1,68 @@
+---
+title: ServiceControl Hosting Options
+summary: Hosting options for running ServiceControl instances
+component: ServiceControl
+reviewed: 2024-12-16
+---
+
+ServiceControl instances can be hosted on:
+
+- Windows Server (virtual machines)
+- Linux containers
+
+
+Deployment options:
+
+- [Monitoring instances](/servicecontrol/monitoring-instances/deployment/)
+- [Audit instances](/servicecontrol/audit-instances/deployment/)
+- [Error instances](/servicecontrol/servicecontrol-instances/deployment/)
+
+## Windows Server
+
+Instances can be installed on Windows Server (virtual) machines using:
+
+- Management UI: ServiceControl Management Utility
+- Commandline: ServiceControl Powershell Module
```

Comment:
We should be downplaying SCMU, as it isn't very devops friendly and promote powershell for Windows more.

Reply:
I've never seen a customer use Powershell. I'm not disagreeing with you and change the order but most of our customers especially those that need support are the point and clicky type users.